src/averageAges.js

```diff
@@ -55,9 +62,35 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = !onlyWithSon
+    ? people.filter(child =>
+      people.find(mother => child.mother === mother.name))
```

Comment:
<img width="889" alt="image" src="https://user-images.githubusercontent.com/102411130/200508538-a64c0120-b2ad-4ea3-b9db-512ec8fc9af1.png">

Reply:
or better use syntax  with braces  for better readability